netmaster/k8snetwork/networkpolicy.go

```diff
@@ -135,204 +249,475 @@ func (k8sNet *k8sContext) deleteEpg(networkname, epgName, policyName string) err
 	}
 
 	for func() error {
-		_, err := k8sNet.contivClient.EndpointGroupGet(defaultTenantName, epgName)
+		_, err := k8sNet.contivClient.
+			EndpointGroupGet(defaultTenantName, epgName)
 		return err
-	}() == nil {
+	}() == nil { //Same as above
 		time.Sleep(time.Millisecond * 100)
 	}
 	return nil
 }
 
-func (k8sNet *k8sContext) createPolicy(policyName string) error {
-	npLog.Infof("create policy %s", policyName)
-
-	if _, err := k8sNet.contivClient.PolicyGet(defaultTenantName, policyName); err == nil {
-		return nil
+//Create policy contiv system
+func (k8sNet *k8sContext) createPolicy(tenantName string,
+	epgName, policyName string) error {
+	//policy := k8sutils.EpgNameToPolicy(epgName, policyName)
+	if _, err := k8sNet.contivClient.
+		PolicyGet(tenantName, policyName); err == nil {
+		npLog.Infof("Policy:%v found contiv", policyName)
```

Comment:
i think instead `get-check-create`, the creating checking operations should be handled on server side, this way will avoid concurrency issue